be/src/pipeline/pipeline_fragment_context.cpp

```diff
@@ -300,7 +300,9 @@ Status PipelineFragmentContext::_build_pipelines(ExecNode* node, PipelinePtr cur
     auto node_type = node->type();
     switch (node_type) {
     // for source
-    case TPlanNodeType::OLAP_SCAN_NODE: {
```

Comment:
seems mysql scan node can be same here ？

Reply:
The current `VMysqlScanNode` inherits from `ScanNode`, and later can unify support for `NewMysqlScanNode` inherits from `VScanNode`